document.txt

```diff
@@ -0,0 +1,8 @@
+This is an important document 
+-------------
+
+Here's an important list:

```

Comment:
@AmyFuq 

---
In reply to: [60946349](https://github.com/kivancmuslu/CFTest/pull/2#discussion_r60946349) [](ancestors = 60946349)